src/vizdoom/src/viz_main.h

```diff
@@ -31,6 +31,9 @@ extern bool vizUpdate;
 extern unsigned int vizLastUpdate;
 extern int vizNodesRecv[VIZ_MAX_PLAYERS];
 
+const int sampling_frequency = 22050;
+const int buffer_len = 1260/(44100/sampling_frequency);
```

Comment:
We pass it elsewhere as a parameter, but here it is hardcoded. This should be unified.
Either hardcode everywhere or use the parameter everywhere.

Reply:
Yea this I'm trying to pass this as an argument from the controller. This is temporary..

Reply:
Now it's fixed, it's an in-game parameter